fdbserver/CommitProxyServer.actor.cpp

```diff
@@ -1123,6 +1142,17 @@ ACTOR Future<Void> assignMutationsToStorageServers(CommitBatchContext* self) {
 				self->toCommit.writeTypedMessage(m);
 				if (SERVER_KNOBS->ENABLE_PARTITIONED_TRANSACTIONS) {
 					self->writeToStorageTeams(pProxyCommitData->keyInfo[m.param1].storageTeams, m);
+					if (self->isSettingUpNewCluster()) {
+						// For a brand new cluster, sending all mutations in txn(version == 1) to seedTeam SS.
+
+						// seedTeam SS need to get the all mutations in txn(version == 1).
+						// because otherwise they would not know they are seedTeam SS until after this transaction,
+						// thus mutations in this txn will be lost.
+
+						// So we write all mutations as normal mutations to private teams of seedTeam servers,
+						// thus all data will be persisted there and mutation lost will be avoided.
+						self->writeToStorageTeams(privateTeamsForServersInSeedTeam, m);
```

Comment:
Same thing should be done in below `m.type == MutationRef::ClearRange` case?

Reply:
do we also have clear range txn in txn v1?

Reply:
There is, for example: https://github.com/apple/foundationdb/blob/176006dff9872d2ffeb5a2fe2af4836bc32f4cb4/fdbserver/masterserver.actor.cpp#L2041.

Even it has no effect, from proxy's perspective, we don't want to make assumptions that are too strong.